djoser/webauthn/views.py

```diff
@@ -95,6 +96,7 @@ def post(self, request, ukey):
 
 class LoginRequestView(APIView):
     permission_classes = (AllowAny,)
+    serializer_class = WebauthnLoginSerializer
 
     def post(self, request):
         serializer = WebauthnLoginSerializer(data=request.data)
```

Comment:
please use `self.serializer_class` instead

Reply:
you mean in class scope? but it has no self

Reply:
No, I mean

```suggestion
        serializer = self.serializer_class(data=request.data)
```

Reply:
Oh, got you! Done!